st2common/st2common/logging/handlers.py

```diff
@@ -37,7 +38,8 @@ def __init__(self, filename, mode='a', maxBytes=0, backupCount=0, encoding=None,
         format_values = {
             'timestamp': timestamp,
             'ts': isotime_str,
-            'pid': pid
+            'pid': pid,
+            'pack': os.path.abspath(os.path.dirname(sys.argv[2])).split('/')[-2]
```

Comment:
I have tested few actions of custom and default packs and the logging is being carried out without any issues . Working on handling the logs for the actions which dont have python code associated with it to use the logger .